### ISSUE TITLE
[Content] Update Parachian RTUs

### ISSUE THREAD

**Issue by NukeManDan** [substrate-developer-hub/substrate-docs#479]
### Content request

https://docs.substrate.io/how-to-guides/v3/parachains/runtime-upgrades/

> it would be really usefull if the HTG also added the srtool bit to identify the codehash since it seems to be the recommended approach

https://github.com/paritytech/srtool

### Are you willing to help with this request?

Yes!

**Comment by chevdor** [comment]
Yes, I can help review and feel free to ping me if you have questions.

**Comment by samelamin** [comment]
I would love to help review, I dont have the capacity to write it just yet though as I myself am not clear how to upgrade
